testdata/baselines/reference/submodule/compiler/declarationEmitUsingTypeAlias1.symbols.diff

```diff
@@ -0,0 +1,14 @@
+--- old.declarationEmitUsingTypeAlias1.symbols
++++ new.declarationEmitUsingTypeAlias1.symbols
+@@= skipped -12, +12 lines =@@
+ === node_modules/some-dep/dist/index.d.ts ===
+ export type OtherType = import('./inner').Other;
+ >OtherType : Symbol(OtherType, Decl(index.d.ts, 0, 0))
+->Other : Symbol(Other, Decl(inner.d.ts, 0, 0))
```

Comment:
Seems like we are regressing in the symbol baselines?

Reply:
Yeah, fix is in latest commits.